pySDC/core/Sweeper.py

```diff
@@ -296,7 +296,13 @@ def func(coeffs):
             self.parallelizable = True
 
         else:
-            raise NotImplementedError(f'qd_type implicit "{qd_type}" not implemented')
+            # see if an explicit preconditioner with this name is available
+            try:
+                QDmat = self.get_Qdelta_explicit(coll, qd_type)
+                self.logger.warn(f'Using explicit preconditioner \"{qd_type}\" on the left hand side!')
+            except NotImplementedError:
+                raise NotImplementedError(f'qd_type implicit "{qd_type}" not implemented')
```

Comment:
How can you make sure that the sweeper respects this? There is no `QE` in the generic implicit sweeper, so this will go unnoticed, right?

Reply:
This is the function for getting `QI` and if you request for instance `EE` as `QI`, that is weird because `EE` is explicit, of course. But the `generic_implicit` sweeper will still work with an explicit preconditioner, it will just solve the systems like `(1-0)u=rhs`, which gives the correct explicit result, as far as I know. Does that answer your question?

Reply:
It does, indeed.